src/DGtal/topology/Object.ih

```diff
@@ -60,7 +61,12 @@
 template <typename TDigitalTopology, typename TDigitalSet>
 inline
 DGtal::Object<TDigitalTopology, TDigitalSet>::Object()
-  : myTopo( 0 ), myPointSet( 0 ), myConnectedness( UNKNOWN )
+  : myTopo( nullptr ),
```

Comment:
I understand that nullptr is the norm for 0 since C++11, while 0 for the norm for NULL before. I am really looking for more stability in C++ !